driver/Driveraux.ml

```diff
@@ -51,12 +58,55 @@ let command ?stdout args =
             argv.(0) fn (Unix.error_message err) param;
     -1
 
+(* This function reimplements quoting of writeargv from libiberty *)
+let gnu_quote arg =
```

Comment:
I'd be tempted to move `gnu_quote` and `diab_quote` to the new lib/Responsefile module.  It makes sense to have parsing and unparsing in the same module.